aea/cli/registry/utils.py

```diff
@@ -0,0 +1,37 @@
+import click
```

Comment:
This could be placed in `aea/cli/registry_api.py` for instance

Reply:
I think it's better to keep this methods isolated for now. Not to create a mess in old CLI module.